### DISCUSSION TITLE
Barcodes?

### DISCUSSION THREAD

**Discussion by blue-j** [scikit-tda/persim#80] [General]
It seems the integration of barcodes has stalled, though PRs have been submitted.  Would love to see it available!  - J

**Comment by catanzaromj** [comment]
It's on the list of things to do, albeit fairly low. Currently, I'm trying to get kepler-mapper's release process updated and ripser's packaging updated. 

One reason for it being low priority is that we have plots available for persistence diagrams already, and the difference between barcodes and persistence diagrams is fairly subjective. There also lots of choices to be made when plotting barcodes that are specific to the problem at hand. For example, how thick should the bars be if there are 5 bars? 50 bars? 5000 bars? I think the approach proposed in https://github.com/scikit-tda/persim/pull/68 is pretty close, and hopefully we can get that integrated soon.

**Reply by blue-j** [reply]
Don't get me wrong, I am very grateful for your shared work and am not demanding anything whatsoever.  I only asked since the PRs were lying around, and I don't know enough to assess their worthiness of committing.
